.yamato/all-hdrp.yml

```diff
@@ -62,6 +62,9 @@ Nightly_HDRP_trunk:
          rerun: on-new-revision
       -  path: .yamato/hdrp-linux-vulkan.yml#HDRP_Linux_Vulkan_editmode_mono_Linear_trunk
          rerun: on-new-revision
+    variables:
+        UTR_VERSION: "current"
```

Comment:
The "current" variable everywhere is just because we forgot to add it in all the jobs when making the PR about this feature, right? It is technically not related to the test filter?

Reply:
Yep correct, forgot to mention that, sorry!